toc.json

```diff
@@ -201,7 +201,7 @@
     "description": "A service which awaits for time to pass, used for system validation.",
     "folder": "services/sleeper/",
     "image": "${DOCKER_REGISTRY}/${DOCKER_IMAGE_NAME}-gpu:${DOCKER_IMAGE_TAG}",
-    "key": "simcore/services/comp/sleeper-gpu",
+    "key": "simcore/services/comp/itis/sleeper-gpu",
```

Comment:
is this where I should be putting all my new services too?

Reply:
Not sure about that, maybe @sanderegg can provide better guidance.
The sleeper was put here for backwards compatibility with the old one, which sits in this namespace.
**Note that the directory in which it is stored does not reflect this path.**

Reply:
@GitHK, @KZzizzle :
Just to make things clear. the "key" entry MUST be the same as the docker image location in the simcore registry. NOTE: I do specify the simcore registry. where the image sits on dockerhub is irrelevant since the simcore platform does not pull from dockerhub.